README.md

```diff
@@ -29,18 +29,29 @@
 
 ### Dependencies
 
-Prerequisites for development:
+Prerequisites on host for development and build:
 
 - python version 3.7
 - pipenv
-- a local postgres (>= 12.0) running
-- postgis extension installed (>= 3.0)
+- `docker` and `docker-compose`
 
-Prerequisite for testing the build/CI stages
+The other services that are used (Postgres with PostGIS extension for metadata and [MinIO](https://www.min.io) as local S3 replacement) are wrapped in a docker compose.
 
-- `docker` and `docker-compose`
+Starting postgres and MinIO is done with a simple
+
+```
+docker-compose up
```

Comment:
It would be good to have `docker-compose up &` part of `make setup` and `docker-compose down` part of `make clean`

Reply:
added it to makefile and readme